### PR TITLE
 community/dnsdist: move from testing

### PR DESCRIPTION
**Pull request by tcely**
Reported the armhf test failures / slowness upstream: https://github.com/PowerDNS/pdns/issues/7590

### CODE REVIEW

File: community/dnsdist/APKBUILD [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Contributor: Vince Mele <vmele@inoc.com>
-# Maintainer: Vince Mele <vmele@inoc.com>
```
Comment (Ikke) [inline community/dnsdist/APKBUILD]: Did you contact Vince about this?

Comment (tcely) [reply community/dnsdist/APKBUILD]: Not since the first PR they submitted. See https://github.com/alpinelinux/aports/pull/3321

**Comment by tcely** [comment]
Would it be possible to get `test-suite.log` from `edge-armhf` so that I can see why the test failed?

**Comment by Ikke** [comment]
@tcely Is this ready to be reviewed / pushed?

**Comment by tcely** [comment]
@Ikke Yes.

I would prefer the DroneCI build didn't time out while testing, but since we didn't get an answer about why the test is slow and I can't increase the time out I can't do more.

This was supposed to be a simple move. Exposing armhf test slowness is beside the point.

**Comment by andypost** [comment]
I takes 10mins in median so looks good

**Comment by tcely** [comment]
@nmeum @Ikke @andypost 

Can this be merged now?

**Pull request closed without merging**